pkg/registry/registry.go

```diff
@@ -40,6 +42,40 @@ type ConfigInfo struct {
 	ExposedPorts *map[string]*interface{} `json:"ExposedPorts"`
 }
 
+// GetDigest returns the pull digest for the image at the registry
+func GetDigest(registry, repository, tag string) (digest.Digest, error) {
```

Comment:
@jLopezbarb 
don't we have a function for this already?

Reply:
We do, but to retrieve the digest that is "callable" the call has to include a special header to the registry. i spotted the same problem when i implemented at the backend https://github.com/okteto/app/pull/2954/files#diff-ed67182f5de738fa9e73e3a19fea061f98446043896d7588c6f2d7f4e7af01a3R36

Reply:
Why do you need it to be "callable"? In any case, is it possible to refactor both functions in a single one?

Reply:
if i set this as tag, the sha does not correspond to the sha used as an image tag, so it can't pull the image.